File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest30728.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest30728 {

    @GetMapping("/BenchmarkTest30728/{pathId}")
    public void BenchmarkTest30728(@PathVariable("pathId") String pathId, HttpServletRequest request, HttpServletResponse response) throws Exception {
        response.sendError(500, "Internal error");
    }
}

File: Benchmarks/enterprise/python/fastapi/testcode/benchmark_test_30728.py
# SPDX-License-Identifier: Apache-2.0
from fastapi import Request
from fastapi import Form


async def BenchmarkTest30728(request: Request, field: str = Form('')):
    field_value = field
    data = '{}'.format(field_value)
    try:
        result = int(str(data))
    except Exception:
        pass
    return {"updated": True}
